### PR TITLE
(727) - Apply - OASys import: Risk Management Plan and Risk To Self Pages

### PR DESCRIPTION
**Pull request by richpjames**
# Context
Similar to #433 #431 #426 we need to import the Risk Management Plan and Risk To Self pages.

[Trello card](https://trello.com/c/PYl2WtOG/727-oasys-risk-and-needs-import)

# Changes in this PR

These are the final two screens, I will put in a subsequent PR to tidy things up a bit

## Screenshots of UI changes

Apply -- allows completion of the form
Apply -- allows completion of the form (4)

### CODE REVIEW

File: integration_tests/tests/apply/apply.cy.ts [hunk 1]
```diff
@@ -696,8 +705,8 @@ context('Apply', () => {
 
       // Then the application should be submitted to the API
       cy.task('verifyApplicationUpdate', application.id).then(requests => {
-        expect(requests).to.have.length(36)
-        const requestBody = JSON.parse(requests[35].body)
+        expect(requests).to.have.length(37)
```
Comment (pezholio) [inline integration_tests/tests/apply/apply.cy.ts:708]: Idly wondering if we can do something about these magic numbers here - is there maybe a way we can count the number of pages or something?

File: integration_tests/tests/apply/apply.cy.ts [hunk 2]
```diff
@@ -696,8 +705,8 @@ context('Apply', () => {
...
+        const requestBody = JSON.parse(requests[36].body)
```
Comment (pezholio) [inline integration_tests/tests/apply/apply.cy.ts:709]: Can probably do this here:

```
const requestBody = JSON.parse(requests[requests.length - 1].body)
```

Comment (pezholio) [reply integration_tests/tests/apply/apply.cy.ts:709]: Also maybe `applicationSubmissionBody` might be a better name, as `requestBody` is a bit generic.

**Review by pezholio — APPROVED** [review]
LGTM - just some tiny things, not necessarily for now. Can probably be included in the next PR if you think they're worth addressing

File: integration_tests/tests/apply/apply.cy.ts (hunk 1 above)
Comment (richpjames) [reply integration_tests/tests/apply/apply.cy.ts:708]: Yep, will fix in a subsequent PR

**Pull request merged**